extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -226,6 +226,20 @@ public function getModalHeaderData() {
 		] );
 	}
 
+	public function getBenefitsModalData() {
+		$memberCount = $this->usersModel->getMemberCount();
+
+		if ( $memberCount < 25 ) {
+			$memberCount = '';
+		} else {
+			$memberCount = $this->wg->Lang->formatNum( $memberCount );
```

Comment:
`$this->getLanguage()->formatNum( $memberCount );` would be preferred here as you have access to the RequestContext.